stackslib/src/chainstate/nakamoto/staging_blocks.rs

```diff
@@ -191,6 +206,28 @@ impl<'a> NakamotoStagingBlocksConnRef<'a> {
         Ok(res.is_some())
     }
 
+    /// Get a staged Nakamoto tenure-start block
+    pub fn get_nakamoto_tenure_start_block(
+        &self,
+        consensus_hash: &ConsensusHash,
+    ) -> Result<Option<NakamotoBlock>, ChainstateError> {
+        let qry = "SELECT data FROM nakamoto_staging_blocks WHERE is_tenure_start = 1 AND consensus_hash = ?1";
+        let args: &[&dyn ToSql] = &[consensus_hash];
```

Comment:
I'm aware; I just do it out of habit because I've found myself on the receiving end of a compiler error where I've expanded the args list of an existing query while forgetting to add this type annotation.